packages/core/src/components/Accordion/Accordion.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import Text from '../Text';
+import { defaultTheme } from '@medly-components/theme';
+import { Preview, Story, Meta, Props } from '@storybook/addon-docs/blocks';
+import { withKnobs, color, select } from '@storybook/addon-knobs';
+import { SimpleAccordion, ThemeInterface } from './Accordion.stories';
+import { Accordion } from './Accordion';
+
+
+<Meta title="Core" component={Accordion} parameters={{ jest: ['Accordion.test.tsx'] }} />
+
+# Accordion
+
+Accordion is a container that can either stand alone or be connected with other components.
+
+<Preview withToolbar>
+    <Story name="Accordion" parameters={{ decorators: [withKnobs] }}>
+       <SimpleAccordion />
```

Comment:
Move the SimpleAccordion code over here only.

Reply:
done